joinmarket/configure.py

```diff
@@ -116,11 +116,16 @@ def jm_single():
 socks5_host = localhost
 socks5_port = 9050
 #for tor
-#host = 6dvj6v5imhny3anf.onion
+#newnym = true
+#newnym_delay = 60
+#tor_host = localhost
+#tor_port = 9051
```

Comment:
added tor_ctrl_* everywhere (_control_ would cause too lines i'm too lazy to break)